ts/screens/wallet/TransactionsScreen.tsx

```diff
@@ -106,19 +125,21 @@ class TransactionsScreen extends React.Component<Props> {
   ) {
     return (
       <React.Fragment>
-        <CardComponent
-          type={"Header"}
-          wallet={selectedWallet}
-          hideFavoriteIcon={false}
-          hideMenu={false}
-          isFavorite={isFavorite}
-          onSetFavorite={(willBeFavorite: boolean) =>
-            handleSetFavourite(willBeFavorite, () =>
-              this.props.setFavoriteWallet(selectedWallet.idWallet)
-            )
-          }
-          onDelete={() => this.props.deleteWallet(selectedWallet.idWallet)}
-        />
+        <View style={styles.cardBox}>
```

Comment:
There is a problem on android
<img src="https://user-images.githubusercontent.com/11773070/105753204-978fcd80-5f48-11eb-8592-2b843256ff72.png" width=300>
You can try to use the same solution of: https://github.com/pagopa/io-app/blob/4e4e1b6640f0d8f996debf3411b86f86456f1030/ts/features/wallet/component/BaseCardComponent.tsx#L63